tools/wptrunner/wptrunner/browsers/base.py

```diff
@@ -280,11 +283,17 @@ def __call__(self, line):
 
 
 class WebDriverBrowser(Browser):
-    __metaclass__ = ABCMeta
-
-    def __init__(self, logger, binary=None, webdriver_binary=None,
-                 webdriver_args=None, host="127.0.0.1", port=None, base_path="/",
-                 env=None, supports_pac=True, **kwargs):
+    def __init__(self,
```

Comment:
The sematics here changed a bit, correct? This is now a concrete class rather than an abstract one. But I don't know that we want that?